convert2rhel/unit_tests/checks_test.py

```diff
@@ -190,179 +190,195 @@ def test_pre_ponr_checks(monkeypatch):
     create_transaction_handler_mock.assert_called_once()
 
 
-@pytest.mark.parametrize(
-    ("convert2rhel_latest_version_test",),
-    ([{"local_version": "0.20", "package_version": "convert2rhel-0:0.22-1.el7.noarch", "pmajor": "7"}],),
-    indirect=True,
-)
-def test_convert2rhel_latest_offline(caplog, convert2rhel_latest_version_test, global_system_info):
-    global_system_info.has_internet_access = False
-    checks.check_convert2rhel_latest()
-
-    convert2rhel_latest_version_test
-    log_msg = "Skipping the check because no internet connection has been detected."
-    assert log_msg in caplog.text
-
+class TestCheckConvert2rhelLatest(object):
+    @pytest.mark.parametrize(
+        ("convert2rhel_latest_version_test",),
+        ([{"local_version": "0.20", "package_version": "convert2rhel-0:0.22-1.el7.noarch", "pmajor": "7"}],),
+        indirect=True,
+    )
+    def test_convert2rhel_latest_offline(self, caplog, convert2rhel_latest_version_test, global_system_info):
+        global_system_info.has_internet_access = False
+        checks.check_convert2rhel_latest()
 
-@pytest.mark.parametrize(
-    ("convert2rhel_latest_version_test",),
-    (
-        [{"local_version": "0.20", "package_version": "convert2rhel-0:0.22-1.el7.noarch", "pmajor": "6"}],
-        [{"local_version": "0.18", "package_version": "convert2rhel-0:1.10-1.el7.noarch", "pmajor": "6"}],
-    ),
-    indirect=True,
-)
-def test_convert2rhel_latest_out_of_date_el6(caplog, convert2rhel_latest_version_test):
-    checks.check_convert2rhel_latest()
+        convert2rhel_latest_version_test
```

Comment:
It seems to me that we are not using this line for anything in the test... 

@abadger What would you like to do about this? Maybe create a Jira issue to take care of this point, or adding another assertion to this test to use this data?

Reply:
You are right.  The line should be removed.  I assume you are wondering whether to remove it now  or open a ticket  because integration tests take so long to run?

Reply:
Yeah, well... If we remove the line, then there is no need for the parametrization in that test, but if we use that for asserting, then we can leave if there, just adjust that line to have an `assert ...`.

Either way, I'm fine with any approach.